data/battle_scripts_1.s

```diff
@@ -8220,27 +8216,30 @@ BattleScript_IntimidateActivatesLoop:
 	jumpifability BS_TARGET, ABILITY_OWN_TEMPO, BattleScript_IntimidatePrevented
 	jumpifability BS_TARGET, ABILITY_OBLIVIOUS, BattleScript_IntimidatePrevented
 .endif
-	statbuffchange STAT_CHANGE_NOT_PROTECT_AFFECTED | STAT_CHANGE_ALLOW_PTR, BattleScript_IntimidateActivatesLoopIncrement
-	jumpifbyte CMP_GREATER_THAN, cMULTISTRING_CHOOSER, 1, BattleScript_IntimidateActivatesLoopIncrement
+BattleScript_IntimidateEffect:
+	statbuffchange STAT_CHANGE_NOT_PROTECT_AFFECTED | MOVE_EFFECT_CERTAIN, NULL
 	setgraphicalstatchangevalues
 	playanimation BS_TARGET, B_ANIM_STATS_CHANGE, sB_ANIM_ARG1
 	printstring STRINGID_PKMNCUTSATTACKWITH
 	waitmessage B_WAIT_TIME_LONG
 	call BattleScript_TryAdrenalineOrb
```

Comment:
`BattleScript_TryAdrenalineOrb` needs BS_SCRIPTING for its string (and maybe more), so before calling it, we need `copybyte sBATTLER, gBattlerTarget`

Reply:
Done :eyes:

I never asked about it, I take it that `sBATTLER` acts as a 1-byte big shortcut to `BS_SCRIPTING` then :eyes:

Reply:
`sBATTLER` is the address in EWRAM of `gBattleScripting.battler` whereas `BS_SCRIPTING` is a caseId to load gBattleScripting.battler in `GetBattlerForBattleScript`